### DISCUSSION TITLE
Extension Methods and Null Propagation

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/csharplang#5902] [Q&A] [answered]
Is there a way to do this:
```
var Value = (Parent?.Child?.GrandChild).ExtensionMethod();
```
Without needing to use the grouping parenthesis?

**Comment by bernd5** [comment]
The grouping parenthesis are not needed. Can you post a concreate sample?

see: https://sharplab.io/#v2:EYLgtghglgdgNAFxAJwK4wD4AEBMBGAWAChiA3CZAAgA9KBeSmAUwHdKANACgEoBuY6gH4AdAAUKTGAhHjkk6WIlTFcqTKUJhAMQD2OnvxJHcHAN7FKlylgDMHSrPm8rxAL7FiWPADZrODgCi1AjmRFbWdl6+WAAslLr6CAAWUADO9tTcoeHuRK5AA==

**Comment by HaloFour** [comment, marked as answer]
You mean you want the extension method to always be called even if the preceding expression results in `null`?  No, the null propagation operator always shortcircuits the remainder of the expression.
